src/main/java/org/folio/s3/client/MinioS3Client.java

```diff
@@ -169,7 +174,7 @@ public String append(String path, InputStream is) {
         .region(region)
         .object(path)
         .stream(is, -1, MAX_PART_SIZE)
-        .extraQueryParams(Map.of("uploadId", uploadId, "partNumber", "2"))
+        .extraQueryParams(Map.of(PARAM_MULTIPART_UPLOAD_ID, uploadId, PARAM_MULTIPART_PART_NUMBER, "2"))
```

Comment:
Curious - are there any plans to use this? -- assuming not --  as I see this method  is marked as deprecated

Reply:
Nope, but I was extracting the constant for the new methods